shared/runtime/gchelper.h

```diff
@@ -41,7 +41,7 @@ typedef uintptr_t gc_helper_regs_t[4];
 typedef uintptr_t gc_helper_regs_t[10];
 #elif defined(__aarch64__)
 typedef uintptr_t gc_helper_regs_t[11]; // x19-x29
-#elif defined(__riscv) && defined(__riscv_xlen) && (__riscv_xlen == 32)
+#elif defined(__riscv) && defined(__riscv_xlen) && ((__riscv_xlen == 32) || (__riscv_xlen == 64))
```

Comment:
Is the `__riscv_xlen` logic needed at all?  Is `#elif defined(__riscv)` enough?  (Is there any other RV arch other than 32 and 64 bits?)

Reply:
There's also [RV128](https://github.com/riscv/riscv-isa-manual/blob/main/src/rv128.adoc), which as you can imagine has 128-bits wide registers.  Can't say I've ever seen that in the wild though.

Reply:
Ah, RV128!  Wow.

I guess the check could then be `__riscv_xlen <= 64`, just to make it a tiny bit simpler.

Reply:
But what if somebody comes up with [RV16](https://dlnext.acm.org/doi/10.1007/s11390-022-0910-x)?  Jokes aside, `__riscv_xlen <= 64` can work, and according to the [RISC-V C API document](https://github.com/riscv-non-isa/riscv-c-api-doc/blob/main/riscv-c-api.md), `__riscv_xlen` must always be defined.  I'll change this bit, thanks for reminding me.